demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
+      - name: source.name
+        value_type: 1 # STRING
+      - name: target.name
+        value_type: 1 # STRING
+      - name: api.name
```

Comment:
Let's remove all the api.X stuff here, we don't have agreement on that stuff anywhere or how to model it.